bin/do-packages-need-a-rebuild.js

```diff
@@ -0,0 +1,31 @@
+const fs = require( 'fs' );
+const path = require( 'path' );
+const digestPackagesDir = require( './calculate-packages-checksum' );
+const chalk = require( 'chalk' );
+
+const checksumFilePath = path.join( __dirname, '..', 'packages', 'packages-checksum.txt' );
+
+// map exit code to answer for less confusion
+const ANSWERS = {
+	YES: 0,
+	NO: 1,
+};
+
+async function shouldRebuild() {
```

Comment:
we should also rebuild if anything in the compilation chain has changed. Different babel version, babel plugin versions, typescript versions, etc etc.

Reply:
a decent proxy for that might be a combo hash of the root yarn.lock and root babel config